packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -105,123 +105,128 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
   }
 
   /**
-   * Creates vault transaction.
+   * Take loan transaction.
    *
-   * @param {CreateVault} createVault txn to create
+   * @param {TakeLoan} takeLoan txn to create
    * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
    * @returns {Promise<TransactionSegWit>}
    */
-
-  async createVault (createVault: CreateVault, changeScript: Script): Promise<TransactionSegWit> {
-    const creationFee = this.network.name === 'mainnet' ? new BigNumber('2') : new BigNumber('1')
-
+  async takeLoan (takeLoan: TakeLoan, changeScript: Script): Promise<TransactionSegWit> {
     return await super.createDeFiTx(
-      OP_CODES.OP_DEFI_TX_CREATE_VAULT(createVault),
-      changeScript,
-      creationFee
+      OP_CODES.OP_DEFI_TX_TAKE_LOAN(takeLoan),
+      changeScript
     )
   }
 
   /**
-   * Create update vault transaction.
+   * PaybackLoan to vault transaction.
    *
-   * @param {UpdateVault} updateVault txn to create
+   * @param {PaybackLoan} paybackLoan txn to create
    * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
    * @returns {Promise<TransactionSegWit>}
    */
 
-  async updateVault (updateVault: UpdateVault, changeScript: Script): Promise<TransactionSegWit> {
+  async paybackLoan (paybackLoan: PaybackLoan, changeScript: Script): Promise<TransactionSegWit> {
     return await super.createDeFiTx(
-      OP_CODES.OP_DEFI_TX_UPDATE_VAULT(updateVault),
+      OP_CODES.OP_DEFI_TX_PAYBACK_LOAN(paybackLoan),
       changeScript
     )
   }
 
   /**
-   * Deposit to vault transaction.
+   * PaybackLoanV2 transaction.
    *
-   * @param {DepositToVault} depositToVault txn to create
+   * @param {PaybackLoanV2} paybackLoanV2 txn to create
    * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
    * @returns {Promise<TransactionSegWit>}
    */
 
-  async depositToVault (depositToVault: DepositToVault, changeScript: Script): Promise<TransactionSegWit> {
+  async paybackLoanV2 (paybackLoanV2: PaybackLoanV2, changeScript: Script): Promise<TransactionSegWit> {
     return await super.createDeFiTx(
-      OP_CODES.OP_DEFI_TX_DEPOSIT_TO_VAULT(depositToVault),
+      OP_CODES.OP_DEFI_TX_PAYBACK_LOAN_V2(paybackLoanV2),
       changeScript
     )
   }
 
   /**
-   * Withdraw from vault transaction.
-   *
-   * @param {withdrawFromVault} withdrawFromVault txn to create
-   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
-   * @returns {Promise<TransactionSegWit>}
-   */
+ * Creates vault transaction.
+ *
+ * @param {CreateVault} createVault txn to create
+ * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+ * @returns {Promise<TransactionSegWit>}
+ * @deprecated vault methods are moving to TxnBuilderVault
+ */
+
```

Comment:
formatting (alignment)

```suggestion
   * Creates vault transaction.
   *
   * @param {CreateVault} createVault txn to create
   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
   * @returns {Promise<TransactionSegWit>}
   * @deprecated vault methods are moving to TxnBuilderVault
 */
```